dmutils/api_stubs.py

```diff
@@ -214,6 +214,7 @@ def brief(status="draft",
         brief['briefs']['applicationsClosedAt'] = "2016-04-07T00:00:00.000000Z"
         brief['briefs']['clarificationQuestionsClosedAt'] = "2016-04-02T00:00:00.000000Z"
         brief['briefs']['clarificationQuestionsAreClosed'] = clarification_questions_closed
+        brief['briefs']['clarificationQuestionsPublishedBy'] = "2016-04-02T00:00:00.000000Z"
```

Comment:
We should probably have all these fields for statuses `awarded`, `cancelled` and `unsuccessful` as well. This code probably dates back to before those statuses existed 🙀

Reply:
Nightmare. Is there somewhere I can find an up to date schema by any chance? 😜

Reply:
Ha! I haven't done schemas for the serializations yet, only the actual DB fields. These dates are all properties inferred from a combo of the `published_at` date and some framework-specific fields in the `data`  JSON field: https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models/main.py#L1655

Reply:
Added